whatlies/embeddingset.py

```diff
@@ -199,7 +213,12 @@ def to_X(self):
         X = emb.to_X()
         ```
         """
-        X = np.array([i.vector for i in self.embeddings.values()])
+        X = np.array(
```

Comment:
Instead, you could have simply used `sklearn.preprocessing.normalize` and write:

```python
from sklearn.preprocessing import normalize

def to_X(self):
    X = np.array([i.vector for i in self.embeddings.values()])
    X = normalize(X) if norm else X
    return X
```